Tasks/UseDotNetV2/task.json

```diff
@@ -151,6 +169,10 @@
         "FailedWhileExtractingPacakge": "Failed while extracting downloaded package with error: %s",
         "InstallingNuGetVersion": "Installing NuGet version %s",
         "FailureWhileInstallingNuGetVersion": "Failed while installing NuGet version. Error: %s",
-        "SettingUpNugetProxySettings": "Setting up proxy configuration for NuGet."
+        "SettingUpNugetProxySettings": "Setting up proxy configuration for NuGet.",
+        "GlobalJsonFound": "Find a global.json at path: %s",
+        "FailedToReadGlobalJson": "The global.json at path: '%s' has the wrong format. For information about global.json, visit here: https://docs.microsoft.com/en-us/dotnet/core/tools/global-json",
+        "VersionNumberHasTheWrongFormat": "The version number: %s doesn't have the correct format.. Valid options are: major.minor.patch || major.minor.patch-previewChannel",
+        "installDotNetCalledWithWrongParameters": "Hey developer you have called the method `` without a `versionSpec` or without `useGlobalJson`. that is impossible."
```

Comment:
Is this not supposed to be user-visible? In that case, you can choose to directly put in the typescript file. Keeping strings in this file results in localization. Also if this is user visible, it can be more professional sounding!

Reply:
no it's an internal exception. I have moved these lines 👍